guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc

```diff
@@ -24,6 +24,21 @@ fdi.pxfactvalue1, fdi.pxfactvalue2 ... fdi.pxfactvalueN::
 fdi.pxauto::
   To set automatic or semi-automatic mode. If set to 0, the image uses semi-automatic mode, which allows you to confirm your choices through a set of dialog options. If set to 1, the image uses automatic mode and proceeds without any confirmation.
 
+fdi.initnet::
+  By default, the image initializes all network interfaces (value `all`). When this setting is set to `bootif`, only the network interface it was network-booted from will be initialized.
+
+fdi.rootpw::
+  By default, root account is locked. Use this option to set root password. Accepts both clear and encrypted passwords.
+
+fdi.ssh::
+  By default, the SSH service is disabled. Set this to `1` or `true` to enable SSH access.
+
+fdi.ipv4.method::
+  By default, NetworkManager IPv4 method setting is set to `auto`. This option overrides it, set it to `ignore` to disable IPv4 stack. This option only works in DHCP mode.
```

Comment:
Is there another word for "stack". Stack seems like one of those words that covers everything and nothing.